R/countBamInGranges.R

```diff
@@ -174,7 +174,6 @@ countBamInGRanges.exomeDepth <- function (bam.file, index = bam.file, granges, m
 #####  second check for consistency between BAM and target regions
   if (sum(! seqs.in.target %in% seqs.in.bam.file)) {  ### if some sequences are missing
     print("Problematic sequences:")
-    print(seqs.in.bam.file)
```

Comment:
Is deze print niet handig voor (error) log?

Reply:
Mogelijk wel...vooralsnog even zo laten want het werkt